shm/Map.hpp

```diff
@@ -6,11 +6,10 @@
 
 constexpr int minPositionXY = 1;
 constexpr int maxPositionXY = 10;
-constexpr int numberOfIslands = 10;
 
 class Map {
 public:
-    Map();
+    Map(size_t numberOfIslands);
```

Comment:
in this task5 weren't we supposed to create constructor without arguments? then create in its body 10 islands?

Reply:
I think that this change makes sense but unfortunately it conflicts with the command in the task.

Reply:
I make it like previously. But we can think about it in future. I will convert it into issue.